### DISCUSSION TITLE
Can I access the slug from a .svelte file?

### DISCUSSION THREAD

**Discussion by 205g0** [sveltejs/kit#8281] [Q&A] [answered]
Pretty much the title, i.e. can I access `slug` from `some/[slug]` within a `.svelte` file without passing it before via `+page.ts` or `+page.server.ts`?

**Comment by AM-77** [comment, marked as answer]
Yes you can, by using the `$page` store from `$app/stores`.

you can access it in +page.svelte or AnyComponent.svelte

```svelte
<script>
  import { page } from "$app/stores";

  $: console.log({ slug: $page.params.slug });
</script>

<p>{$page.params.slug}</p>

```

**Reply by 205g0** [reply]
Thanks, that's neat!
